app/views/reservations/edit.html.haml

```diff
@@ -25,6 +25,13 @@
 
   = render "reservation_fields", f: f
 
+  .well
+    .container
+      .row
+        .span
+          %h4= OrderDetail.model_name.human
```

Comment:
This shouldn't cause any confusion. `admin_note` is only used for admin and offline reservations. The order detail note is for user orders.